src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -168,10 +168,21 @@ Just set the `pinned` prop.
 				:force-menu="forceMenu"
 				:default-icon="menuIcon"
 				@update:open="onMenuToggle">
-				<ActionButton v-if="editable && !editingActive" icon="icon-rename" @click="handleEdit">
+				<template #icon>
+					<!-- @slot Slot for the custom menu icon -->
+					<slot name="menu-icon" />
+				</template>
+				<ActionButton v-if="editable && !editingActive" @click="handleEdit">
+					<template #icon>
+						<Pencil :size="20" decorative />
+					</template>
 					{{ editLabel }}
 				</ActionButton>
-				<ActionButton v-if="undo" icon="app-navigation-entry__deleted-button icon-history" @click="handleUndo" />
+				<ActionButton v-if="undo" icon="app-navigation-entry__deleted-button" @click="handleUndo">
+					<template #icon>
+						<Undo :size="20" decorative />
```

Comment:
This is not decorative?
The action has no text, title or aria label either, so it's not accessible

Reply:
Then let's provide an aria label to the `ActionButton`s.

Reply:
I added aria labels to the buttons now (although this doesn't really has something to do with replacing the icons, though).
However, the translations need to be updated, which I cannot do at the moment (no npm at hand). Any volunteers?